.github/workflows/go.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           - 1.12.x
           - 1.13.x
           - 1.14.x
+          - 1.15.x
```

Comment:
we should wait a little for this change?

Reply:
Intention was Go1.15 is now a deprecated version of Go. Not necessarily deprecated by the SDK. This section is for Go versions that are deprecated by the language maintainers.